File: backend/hot-place-service/src/main/java/com/hotspot/hotplace/dto/HotPlaceResDto.java
package com.hotspot.hotplace.dto;

import com.hotspot.hotplace.entity.HotPlaceType;
import lombok.Data;

@Data
public class HotPlaceResDto {

    private Long id;

    private String placeName;

    private String addressName;

    private String roadAddressName;

    private String phone;

    private double x;

    private double y;

    private String category;

    private HotPlaceType placeType;

    private int likeCount;

    private int visitorCount;

    private int rank;
}

File: backend/member-service/src/main/kotlin/com/hotspot/global/eureka/dto/HotPlaceResDto.kt
package com.hotspot.global.eureka.dto

class HotPlaceResDto(
    val id: Long,
    val placeName: String,
    val addressName: String,
    val roadAddressName: String,
    val phone: String,
    val x: Double,
    val y: Double,
    val category: String,
    val placeType: HotPlaceType,
    val likeCount: Int,
    val visitorCount: Int,
)